index.bs

```diff
@@ -1105,20 +1140,28 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
           return a Promise rejected with a newly created {{TypeError}}.
         </li>
         <li>
-          Let <var>p</var> be a new promise.
+          Let <var>p</var> be a new Promise.
         </li>
         <li>
-          <a>In parallel</a>, run the <a>Create a MediaCapabilitiesEncodingInfo</a>
-          algorithm with <var>configuration</var> and resolve <var>p</var>
-          with its result.
+          Return <var>p</var>.
```

Comment:
Same comment as above re. ordering of steps.

Reply:
Done